spec/paperclip/io_adapters/attachment_adapter_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 describe Paperclip::AttachmentAdapter do
   before do
-    rebuild_model path: "tmp/:class/:attachment/:style/:filename", styles: {thumb: '50x50'}
+    rebuild_model path: "tmp/:class/:attachment/:style/:filename",
+      styles: {thumb: '50x50'}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.